cluster-autoscaler/simulator/cluster.go

```diff
@@ -168,6 +181,7 @@ func calculateUtilizationOfResource(node *apiv1.Node, nodeInfo *schedulercache.N
 	nodeAllocatable, found := node.Status.Allocatable[resourceName]
 	if !found {
 		return 0, fmt.Errorf("failed to get %v from %s", resourceName, node.Name)
+		return 0, fmt.Errorf("Failed to get %v from %s, resource is not ready yet", resourceName, node.Name)
```

Comment:
Seems like you added a new line, instead of modifying the existing one.

Reply:
Removed.